src/lib/components/Popover.svelte

```diff
@@ -98,6 +100,11 @@
     border-radius: var(--border-radius);
 
     box-shadow: var(--overlay-box-shadow);
+
+    &.noMaxWidth {
+      // 100vw - right padding
+      max-width: calc(100vw - var(--padding));
```

Comment:
Do we actually need `max-width: var(--size);` and if yes, do you remember why?

If we don't need it, maybe it should always be `max-width: calc(100vw - var(--padding));`?

I generally like to avoid JS to apply a UI behavior on a single CSS element, I would rather user a CSS variable, so just try to understand if there is not another way.

Reply:
Because of [this popover](https://github.com/dfinity/nns-dapp/pull/2764) the fixed max-width should be still supported.
<img width="134" alt="image" src="https://github.com/dfinity/gix-components/assets/98811342/e66c6aef-2009-41f3-9faa-f028382cfbc3">

Reply:
Thanks for the feedback